openqabot/approver.py

```diff
@@ -132,6 +132,32 @@ def is_job_marked_acceptable_for_incident(self, job_id: int, inc: int) -> bool:
             pass
         return False
 
+    @lru_cache(maxsize=512)
+    def validate_job_qam(self, job: int) -> bool:
+        # Check that valid test result is still present in the dashboard (see https://github.com/openSUSE/qem-dashboard/pull/78/files) to avoid using results related to an old release request
+        try:
+            qam_data = get_json("api/jobs/" + str(job), headers=self.token)
+        except Exception as e:  # pylint: disable=broad-except
+            log.info(
+                "Cannot reach the dashboard database to check if job %s is still valid",
+                job,
+            )
+            log.exception(e)
+            return False
+        if "error" in qam_data:
+            log.info(
+                "Cannot find job %s in the dashboard database to make sure it is valid",
+                job,
+            )
+            return False
+        if qam_data["status"] != "passed":
+            log.info(
+                'Job %s does not appear "passed" in the qam-dashboard database',
```

Comment:
what do you mean with "appear" here? Why not just "is not recorded as passed"?

Reply:
Sure, will change!